docs/prow/developers-faq.md

```diff
@@ -0,0 +1,51 @@
+# Developers FAQ
+Frequently Asked Questions (FAQ) a developer can have, regarding working with Prow as a Continuous Integration (CI) mechanism.
+
+**Q: I would like to add a new component/item and have it operated by Prow**
+
+**A:** Creating a CI pipeline for a new component requires a new ProwJob. Please see [this document](create-component-jobs.md) for further information
+
+---
+**Q: I need my component to be a part of a release, how to do it?**
+
+**A:** A release job is a special kind of ProwJob that uses a specific release branch. Please see [this document](create-release-jobs.md) for further information
+
+---
+**Q: I would like to use Prow to check something on my fork, how should I do it?**
```

Comment:
Rename this question to something like: "I want to test my changes made in scripts defined in the test-infra repository. How to do that?"